server/src/model.ts

```diff
@@ -869,7 +869,13 @@ export async function getAssignmentDataFromCode(
         isAssigned: true,
         isFolder: false,
       },
-      include: {
+      select: {
+        id: true,
+
+        // TODO: these values are only used in tests. Do we need them in this API?
```

Comment:
Go ahead and delete this fields from the query and delete the lines from the test checking them.